examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -107,5 +278,42 @@ setupShouldPurchasePromoProductListener = function() {
   });
 },
 
+setupPurchaseButtons = function () { 
+  const prototypeButton = document.getElementById("prototype-button");
+  const parentNode = prototypeButton.parentNode;
+  Purchases.getOfferings(
+    offerings => {
+      const availablePackages = offerings.current.availablePackages;
+      availablePackages.forEach(package => {
+        var purchaseButton = prototypeButton.cloneNode(true);
+        purchaseButton.hidden = false;
+        purchaseButton.id = package.product.identifier;
+        purchaseButton.textContent = "Buy " + package.identifier + package.product.priceString;
```

Comment:
Does this need to be separated with a space? 
```suggestion
        purchaseButton.textContent = "Buy " + package.identifier + " " + package.product.priceString;
```